test_interpreter.py

```diff
@@ -486,112 +475,41 @@ def test_phi02():
     argc = 1
     argv = ["temp.bc"]
     result = interp.run_bytecode(argc, argv, r"""
-; ModuleID = 'test.bc'
-target datalayout = "e-m:e-i64:64-f80:128-n8:16:32:64-S128"
-target triple = "x86_64-unknown-linux-gnu"
 
 @.str = private unnamed_addr constant [7 x i8] c"%d %d\0A\00", align 1
-@.str.1 = private unnamed_addr constant [5 x i8] c"z %d\00", align 1
 
-; Function Attrs: nounwind uwtable
-define i32 @main(i32 %argc, i8** nocapture readnone %argv) #0 {
+define i32 @main(i32 %argc, i8** %argv) {
 entry:
-  br label %for.body
+  %x.addr = alloca i32, align 4
```

Comment:
Does this program run as expected when you compile it using the llvm tools, btw?